core/src/main/scala/sjsonnew/AdditionalFormats.scala

```diff
@@ -97,4 +97,20 @@ trait AdditionalFormats {
     }
   }
 
+  /**
+   * A `JsonFormat` that doesn't write and always return this instance of `a`.
+   */
+  def asSingleton[A](a: A): JsonFormat[A] = new JsonFormat[A] {
+    def read[J](jsOpt: Option[J], unbuilder: Unbuilder[J]): A = a
+    def write[J](obj: A, builder: Builder[J]): Unit = ()
+  }
+
+  /**
+   * A `JsonFormat` that can read and write an instance of `T` by using a `JsonFormat` for `U`.
+   */
+  def wrap[T, U](f1: T => U, f2: U => T)(implicit fu: JsonFormat[U]): JsonFormat[T] = new JsonFormat[T] {
```

Comment:
I like the name `project` like in Shapeless - https://github.com/milessabin/shapeless/blob/shapeless-2.3.1/core/src/main/scala/shapeless/typeclass.scala#L42